src/errors.js

```diff
@@ -0,0 +1,18 @@
+import dedent from 'dedent';
+import { render as renderTemplate } from 'mustache';
+import CONSTANTS from './constants';
+
+export default {
+    mainUrlWasNotLoaded: dedent `
+        A page in the main window specified by the mainWindowUrl option was not loaded. 
+        The affected page is one of the following:
```

Comment:
It's better but I think we should say where you should specify it (via the mainWindowUrl option)